### ISSUE TITLE
Callout example not working for language fsharp/fs

### ISSUE THREAD

**Issue by jannikbuschke** [code-hike/codehike#420]
example: https://stackblitz.com/edit/github-wxcn1t?file=app%2Fpage.mdx

thanks for this awesome work!, looking forward to use it

**Comment by ogizanagi** [comment]
Hello 👋🏻 

Neither it seems to be parsing any annotations for graphql blocks

    ```graphql
    # !callout[/id/] This is a callout
    query foo {
      foo { id }
    }
    ```

**Comment by pomber** [comment]
Thanks. Both are probably issues with https://github.com/code-hike/lighter. I need to make some changes there soon, so I'll make sure to fix this too.

**Comment by pomber** [comment]
`rust` also failing

````
```rust
// !mark(1:2)
fn main() {
    let s1 = String::from("hello");
    let s2 = s1;

    println!("{}", s1);
    println!("{}", s2);
}
```
````

From https://github.com/code-hike/v1/issues/14

**Comment by pomber** [comment]
Should be fixed in v1.0.0-beta.2

**Comment by jannikbuschke** [comment]
Confirmed for fsharp/fs 🎉
